### DISCUSSION TITLE
Are coordinates of a pointcloud somehow rounded or sampled when visualizing?

### DISCUSSION THREAD

**Discussion by cmosig** [isl-org/Open3D#4997] [Q&A] [answered]
When I plot the xy coordinates in matplotlib I get the following result:

![image](https://user-images.githubusercontent.com/32590522/163130017-ba74550d-9010-4611-a7ba-dbb957805623.png)

But when I use open3d and turn the camera such that I look from the top, so effectively the same view as above, then I get the following result:
![image](https://user-images.githubusercontent.com/32590522/163130235-a0f85e56-3825-41c6-a408-0f66a5fd9c47.png)

There is clearly some form of rounding in play here. I have searched the Internet and forum but just can't figure out what is causing this issue.

**Comment by dancergraham** [comment, marked as answer]
Hello, this looks like an issue with rounding to float32 somewhere in open3d - a workaround could be to move your points close to zero before visualising them.

**Reply by cmosig** [reply]
Yes, you are right, this is the issue. I moved the points closer to zero and now it looks great. A warning by open3d would be nice though.
